scripts/install-kubectl.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Platform check
+PLATFORM=`uname -s | awk '{print tolower($0)}'`
+echo "Your platform is $PLATFORM" 
+
+echo -e "\nGet kubectl binary\n"
+# Platform check for kubectl binary
+PLATFORM=`uname -s | awk '{print tolower($0)}'`
+# Download and install kubectl
+curl -sSL -o bin/kubectl https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/$PLATFORM/amd64/kubectl
+
+# Make the kubectl executable 
+chmod +x kubectl
```

Comment:
It seems you download `kubectl` to `bin/kubectl`, but the `chmod` runs on `./kubectl`.

Reply:
> It seems you download `kubectl` to `bin/kubectl`, but the `chmod` runs on `./kubectl`.

Ahh...my bad I missed the path. Done, thanks.

Reply:
For testing, should we lock in a stable version of kubectl?

Reply:
> For testing, should we lock in a stable version of kubectl?

Yes, we can do that, however one thing comes to mind that each time we need to be aware of a static kubectl version that we want to update which will increase the maintenance cost. I think we should keep a static version if its a hard requirement for certain functionalities but not sure which version to keep here. @wtam2018 can you please suggest the kubectl version
I think we we need to keep the argocd cli version static too ping @amitkrout @dewan-ahmed

Reply:
> I think we we need to keep the argocd cli version static too ping @amitkrout @dewan-ahmed 

@jannfis Is argocd CLI release along with the argocd server ? for example OCP release has its parallel oc cli release like ocp 4.7 has oc 4.7 cli release. If yes then we can consume the static argo cli version which matches the argo server version we consume in gitops operator.

Reply:
We can set to the current kubectl version:  v1.20.5.  Locking in a version as opposed to floating version helps to produce repeatable test results.    I would lock the argocd cli version as well.   We should know/control the tool versions that we are testing with.

Reply:
> We can set to the current kubectl version: v1.20.5. Locking in a version as opposed to floating version helps to produce repeatable test results. I would lock the argocd cli version as well. We should know/control the tool versions that we are testing with.

Sounds good.
I will lock kubectl version in this pr and will create a separate pr to lock argocd cli version